.gitignore

```diff
@@ -22,6 +22,9 @@ lerna-debug.log
 .settings/
 *.sublime-workspace
 .history/*
+.vscode
+# For prettier
+!.vscode/settings.json
```

Comment:
We have this same line a little lower in the file, was that not working?

Reply:
Yep, I'm blind. I'll remove this.

Reply:
Fixed in latest commit.